builtin/mainmenu/tab_online.lua

```diff
@@ -174,6 +174,13 @@ local function get_formspec(tabview, name, tabdata)
 				"server_view_clients.png") .. ";btn_view_clients;]"
 		end
 
+		if selected_server.url then
+			retval = retval .. "tooltip[btn_server_url;" .. fgettext("Server URL") .. "]"
```

Comment:
```suggestion
			retval = retval .. "tooltip[btn_server_url;" .. fgettext("Server Website") .. "]"
```

I think it's better for communication

Reply:
I agree. Might be easier for translation too...

Reply:
IMO it would also be nice to display the URL in the tooltip.

Reply:
Ooops, I was too quick. Mhm, don't know the url is already shown in the dialog.